RIOT-OS/RIOT
# pkg/lora-serialization: bump to latest version and update test for AVR

### Contribution description

This PR updates the lora-serialization package to the latest version which contains a fix to make it more compliant with non 32bit architectures.

This PR is also adapting the corresponding test for AVR platform: on AVR doubles are simply represented as float. Thus the values used for GPS in the test 02 are not encoded the same way on this platform.

There's also a small commit that changes a couple of cpp style comments into C-style comments.




### Testing procedure

The following commands should work:
```
$ make -C tests/pkg-lora-serialization all test
$ make BOARD=samr21-xpro -C tests/pkg-lora-serialization flash test
$ make BOARD=atmega256rfr2-xpro -C tests/pkg-lora-serialization flash test
```




### Issues/PRs references

Tick the lora-serialization item in #12651
